app/src/config/development.toml

```diff
@@ -0,0 +1,50 @@
+[docker]
+host = "localhost"
+
+[core]
+platformOriginUrl = ""
+streamrUrl = ""
+
+[client]
+graphUrl = ":8000/subgraphs/name/streamr-dev/network-contracts"
+restUrl = "/api/v1"
+storageNodeAddress = "0xFCAd0B19bB29D4674531d6f115237E16AfCE377c" # STREAMR_ENGINE_NODE_ADDRESSES
+
+[client.mainchain]
+name = "dev_ethereum"
+dataTokenAddress = "0xbAA81A0179015bE47Ad439566374F2Bae098686F"
+chainId = 1
+
+[client.mainchain.rpc]
+url = ":8545"
```

Comment:
FYI

- empty string → `http://localhost`
- `:PORT` → `http://localhost:PORT`
- `/path` → `http://localhost/path`

See `formatConfigUrl` for details.

Reply:
`localhost` is `docker.host` so it's very easy to develop with docker on a different machine. Production links don't match regexps so they're as they are. `formatConfigUrl` respects the `STREAMR_DOCKER_DEV_HOST` env variable, too.

Reply:
I'm drifting away from `.env.required` and `.env.template` files in this PR. Don't like duplications they introduce.

I'll replace them with proper pre-build validations and tests.